api/odk/helpers/aggregate-submissions.js

```diff
@@ -5,9 +5,34 @@ var async = require('async');
 
 var ASYNC_LIMIT = 10;
 
-module.exports = function (formName, errorCallback, aggregateCallback) {
+module.exports = function (req, errorCallback, aggregateCallback) {
+    var formName = req.params.formName;
+    var limit = parseInt(req.query.limit);
+    var offset = req.query.offset;
+
+    // default to 100 for limit
+    if (isNaN(limit) || !Number.isInteger(limit) || limit < 1) {
```

Comment:
When will `Number.isInteger` be false? I think `parseInt` either returns a number or `NaN`.